go.mod

```diff
@@ -189,6 +193,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	gopkg.in/zorkian/go-datadog-api.v2 v2.29.0
 	gotest.tools v2.2.0+incompatible
+	honnef.co/go/tools v0.1.1 // indirect
```

Comment:
There are a lot of `// indirect` dependencies added here that I don't think we want, can you double check?